dashboard/src/components/AppView/AppView.tsx

```diff
@@ -100,21 +111,32 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
       }, {});
     this.setState({ otherResources });
 
-    const deployments = manifest.filter(d => d.kind === "Deployment");
-    const services = manifest.filter(d => d.kind === "Service");
-    const ingresses = manifest.filter(d => d.kind === "Ingress");
     const sockets: WebSocket[] = [];
-    for (const d of deployments) {
-      sockets.push(this.getSocket("deployments", d.apiVersion, d.metadata.name, newApp.namespace));
-    }
-    for (const svc of services) {
-      sockets.push(this.getSocket("services", svc.apiVersion, svc.metadata.name, newApp.namespace));
-    }
-    for (const i of ingresses) {
-      sockets.push(this.getSocket("ingresses", i.apiVersion, i.metadata.name, newApp.namespace));
-    }
+    let secrets = {};
+    manifest.forEach((i: IResource | ISecret) => {
```

Comment:
I don't think the type `ISecret` is needed here since we convert it later, but perhaps I am missing something?

Reply:
It's needed to avoid this issue:

```
Conversion of type 'IResource' to type 'ISecret' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Property 'data' is missing in type 'IResource'.
```